tests/integration/test_vid_club.py

```diff
@@ -23,23 +29,20 @@ def json():
 
 
 @pytest.mark.init
-def test_create_club_class():
-    vc = VidClub()
-    assert vc
+def test_default_credential_param():
+    assert vc.credentials != None and type(vc.credentials) == dict
 
 
 @pytest.mark.init
-def test_default_credential_param():
-    vc = VidClub()
-    assert vc.credentials != None and type(vc.credentials) == dict
+def test_create_club_class():
+    assert vc
 
 
 @pytest.mark.proper
 def test_build_query_wrong_source():
     with pytest.raises(
         ValidationError, match=r"Pick one these sources: jobs, product, company, survey"
     ):
```

Comment:
Please add docstring to test function that will shortly summarize what they are testing